src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +65,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() { return false; }
```

Comment:
remember the Animal, Cat and Dog examples we talked about in tutorial? Should there be an implementation for this?

Reply:
should add comment for this as well